src/chatty/CustomPaths.java

```diff
@@ -74,21 +74,10 @@ public void updateFromSettings(PathType type) {
 
     public synchronized void setCustom(Chatty.PathType type, Path path, String info, boolean requireExists) {
         ChattyPath p = get(type);
-        if (!path.isAbsolute()) {
-            path = Paths.get(System.getProperty("user.dir"), path.toString());
-        }
-        if (requireExists && !Files.isDirectory(path)) {
-            p.customDir = null;
-            p.invalidCustomDir = path.toString();
-            p.customInfo = info;
-        }
-        else {
-            p.customDir = path;
-            p.invalidCustomDir = null;
-            p.customInfo = info;
-        }
+        p.setCustomValuesForObject(path,info,requireExists);
```

Comment:
Existing code uses a space after comma for passing parameters.
```suggestion
        p.setCustomValuesForObject(path, info, requireExists);
```